FindOnPage.md

```diff
@@ -0,0 +1,382 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API provides methods and events to support text finding and navigation within a WebView2 control. 
+It allows developers to initiate find operations, navigate between find results, and customize various find configurations.
+
+## Examples
+
+
+#### Description
+To initiate a find operation within a WebView2 control, developers can utilize the `StartFindOnPage` method. 
+This method allows specifying the find term and configuring other find parameters using the `ICoreWebView2FindConfiguration` interface.
+
+#### Create/Specify a Find Configuration
+
+
+
+```cpp
+ICoreWebView2FindConfiguration CreateFindConfiguration()
+{
+    ICoreWebView2Environment5 environment = webView2Control.GetEnvironment();
+    ICoreWebView2FindConfiguration configuration;
+    environment.CreateFindConfiguration(out configuration);
+    configuration.FindTerm = "example";
+    configuration.FindDirection = COREWEBVIEW2_FIND_DIRECTION.COREWEBVIEW2_FIND_DIRECTION_FORWARD;
+    configuration.IsCaseSensitive = false;
+    configuration.ShouldMatchWord = false;
+    return configuration;
+}
+```
+
+### Start a Find Operation
+
+
+```cpp
+//! [StartFindOnPage]
+void AppWindow::StartFindOnPage(const std::wstring& findTerm)
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Create find configuration
+    auto configuration = CreateFindConfiguration();
+
+    // Start the find operation
+    find.StartFind(configuration, OnFindOperationCompleted);
+}
+//! [StartFindOnPage]
+```
+### Stop an existing find operation
+#### Description
+To stop an ongoing find operation within a WebView2 control, developers can use the `StopFind` method of the `ICoreWebView2Find` interface.
+```cpp
+//! [StopFind]
+void AppWindow::StopFind()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+    
+    // Stop the find operation
+    find.StopFind();
+}
+//! [StopFind]
+```
+
+### Retrieve the Number of Matches
+
+#### Description
+To retrieve the total number of matches found during a find operation within a WebView2 control, developers can utilize the `GetMatchCount` method.
+
+
+```cpp
+//! [GetMatchCount]
+void AppWindow::GetMatchCount()
```

Comment:
Unfortunately, StartFind completing, and the match counts/ match index being reported are not tied together. So if you call either of them within the FindOperationCompletedHandler, it will not return the most updated values.